data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES ('Agumon', '2020-05-14', 0, true, 10.23);
```

Comment:
- [ ] The date of birth for `Agumon` is not matching the give date of birth. Kindly fix this.

Given DOB
![image](https://user-images.githubusercontent.com/81932478/235502662-f9f1e863-34c1-4c09-a719-c1882369ad7f.png)